coordinator/core/marbleapi.go

```diff
@@ -107,8 +107,29 @@ func (c *Core) Activate(ctx context.Context, req *rpc.ActivationReq) (*rpc.Activ
 		SealKey:    Secret{Public: sealKey, Private: sealKey},
```

Comment:
Not directly related to this PR, but the function is a bit long. Can you refactor the code from
privk, err := ecdsa.GenerateKey(elliptic.P256(), rand.Reader)
till
authSecrets := reservedSecrets{
into a new function?